### DISCUSSION TITLE
Game updates to be done in-game instead of having to redownload

### DISCUSSION THREAD

**Discussion by Minorsonek** [ppy/osu#21703] [Ideas] [answered]
Hi, so I was wondering, are there any plans to get in-game updates for Mac users? Osu is getting updates quite frequently, and it is so frustrating when you open the game, see that there is an update, then you have to close it, go to github, download and stuff, manually replace in applications folder, then reopen again - I think a nice quality of life feature would be to automate it. 

Specifically talking about Mac version for apple silicon, I don't know how is the situation for other versions

**Comment by frenzibyte** [comment, marked as answer]
There'll be eventual support for in-game updates on macOS, the updater we rely on looks to have already added support for macOS, so we're not far away form there.

**Reply by Minorsonek** [reply]
Cool, good news. Perhaps can I do anything to help make it available sooner?

**Reply by peppy** [reply]
Unless you are able to implement yourself (will require some good knowledge of the ecosystem), then there's not much to be done.

If you're looking for an alternative there's a brew package that is updated automatically.
